modules/customeradd.php

```diff
@@ -93,6 +93,11 @@
 	if($customeradd['name'] == '' && !$customeradd['type'])
 		$error['name'] = trans('First name cannot be empty!');
 	
+	if (ConfigHelper::getConfig('phpui.add_customer_group_required', false)) {
```

Comment:
"Opakuj" to sprawdzenie jeszcze w ConfigHelper::checkValue(...), żeby nie było sytuacji, że ktoś ustawi 'bleble' i będzie to dziwnie traktowane.